tools/wizard_spell_converter/spell_converter.json

```
@@ -0,0 +1,74 @@
+{
+	"HS": "Mastercrafted Armor Set",
+	"GFW": "Greater Force Wall",
+
+	"BMAC": "Battlemage Armour Charges",
+	"STD": "Spacetime Distortion",
+	"CHG": "Charge",
+	"BMA": "Battlemage Armour",
+	"BOS": "Bottle of Shadows",
+	"SHH": "Silencio",
+	"SGH": "Summon Ghosts",
+	"SLG": "Silence Ghosts",
+	"BD": "Blind",
+	"LB": "Lightning Bolt",
+	"CC": "Curse of the Cluwne",
+	"BT": "Banana Touch",
+	"MI": "Mime Malaise",
+	"HH": "Curse of the Horseman",
+	"DG": "Disintegrate",
+	"FB": "Fireball",
+	"FS": "Flesh to Stone",
+	"MU": "Mutate",
+	"RF": "Rod Form",
+	"IG": "Lesser Summon Guns",
+	"DT": "Disable Tech",
+	"FW": "Force Wall",
+	"RS": "Rathen''s Secret",
+	"RP": "Repulse",
+	"SM": "Smoke",
+	"LD": "Bind Soul",
+	"MM": "Magic Missile",
+	"TS": "Time Stop",
+	"SF": "Sacred Flame and Fire Immunity",
+	"KN": "Knock",
+	"BL": "Blink",
+	"EJ": "Ethereal Jaunt",
+	"GK": "Greater Knock",
+	"MT": "Mindswap",
+	"TP": "Teleport",
+	"IS": "Summon Item",
+	"NC": "Remove Clothes Requirement",
+	"SG": "Summon Guns",
+	"SU": "Summon Magic",
+	"NS": "A Necromantic Stone",
+	"SO": "Scrying Orb",
+	"SS": "Six Soul Stone Shards and the spell Artificer",
+	"WA": "Wand Assortment",
+	"CH": "Cursed Heart",
+	"VG": "Voice of God",
+	"WC": "Warp Cubes",
+	"EM": "Everfull Mug",
+	"MJ": "Mjolnir",
+	"SI": "Singularity Hammer",
+	"CK": "Cursed Katana",
+	"SB": "Spellblade",
+	"MH": "Meat hook",
+	"SD": "Staff of Door Creation",
+	"SH": "Staff of Healing",
+	"SL": "Staff of Slipping",
+	"SA": "Staff of Animation",
+	"ST": "Staff of Change",
+	"SC": "Staff of Chaos",
+	"BO": "Bottle of Ooze",
+	"HB": "Bottle of Tickles",
+	"BB": "Bottle of Blood",
+	"RB": "Bottle of Ectoplasm",
+	"CT": "Contract of Apprenticeship",
+	"TD": "Guardian Deck",
+	"OM": "Offense Focus : Mutant",
+	"DL": "Defense Focus : Lich",
+	"UW": "Utility Focus : Wands",
```

While you're here, I think this would look better.
```suggestion
	"OM": "Offense Focus - Mutant",
	"DL": "Defense Focus - Lich",
	"UW": "Utility Focus - Wands",
```

Just don't forget to change the DM definition as well.